src/screens/TripDetails/QuayDepartures/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {View, Animated, RefreshControl} from 'react-native';
+import {LocationWithMetadata} from '@atb/favorites/types';
+import ScreenHeader from '@atb/components/screen-header';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import DeparturesList from '@atb/components/departure-list/DeparturesList';
+import {useDepartureData} from '@atb/screens/Nearby/state';
+import {NavigationProp, RouteProp} from '@react-navigation/native';
+import {DetailsStackParams} from '@atb/screens/TripDetails';
+
+export type QuayDeaparturesNavigationProps = NavigationProp<DetailsStackParams>;
+export type QuayDeparturesRouteProp = RouteProp<
+  DetailsStackParams,
+  'QuayDepartures'
+>;
+
+export type QuayDeparturesRouteParams = {
+  location: LocationWithMetadata;
+};
+
+type RootProps = {
+  route: QuayDeparturesRouteProp;
+  navigation: QuayDeaparturesNavigationProps;
+};
+
+const QuayDepartures: React.FC<RootProps> = ({route}) => {
+  const styles = useNearbyStyles();
+  const {theme} = useTheme();
+  const {top: paddingTop} = useSafeAreaInsets();
+
+  const {state: departureState, refresh} = useDepartureData(
+    route.params.location,
+  );
+
+  const {data: departureData, isLoading} = departureState;
+
+  return (
+    <View style={styles.screen}>
+      <View style={[styles.header, {paddingTop}]}>
+        <ScreenHeader
+          leftButton={{type: 'back'}}
+          title={route.params.location.name}
+        />
+      </View>
+      <Animated.ScrollView
```

Comment:
Is there a reason for this being animated scroll view?

Reply:
yes, the Animated.scrollView has a refreshControl option that lets us use the pull-down-to-refresh feature

Reply:
Regular [`<ScrollView>` should also have that](https://reactnative.dev/docs/scrollview#refreshcontrol)

Reply:
You are indeed correct